arangod/Utils/Events.cpp

```diff
@@ -33,6 +33,7 @@ void Authenticated(GeneralRequest const&, rest::AuthenticationMethod) {}
 void NotAuthorized(GeneralRequest const&) {}
 void CreateCollection(std::string const& db, std::string const& name, int result) {}
 void DropCollection(std::string const& db, std::string const& name, int result) {}
+void PropertyUpdateCollection(std::string const& collectionName, VPackSlice const& propertiesSlice) {}
```

Comment:
The collection name alone may not be useful when there are multiple database.
We should add the database name here too (similar to `TruncateCollection`, `DropCollection`, `CreateCollection`).